assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -189,9 +245,13 @@ export default function UserInputQuestionnaire() {
 
 			{ activeSlug === 'preview' && (
 				<UserInputPreview
+					isNavigating={ isNavigating }
+					isSavingSettings={ isSavingSettings }
+					error={ error }
+					submitChanges={ submitChanges }
+					goToPreview={ goToPreview }
```

Comment:
@benbowler the `goToPreview` property isn't needed for the `UserInputPreview` component. Let's get rid of it.